src/V3EmitCSyms.cpp

```diff
@@ -773,6 +774,7 @@ void EmitCSyms::emitSymImp() {
                     bounds += ",";
                     bounds += cvtToStr(basicp->lo());
                     pdim++;
+                    pwidth *= basicp->hi() - basicp->lo() + 1;
```

Comment:
```suggestion
                    pwidth *= basicp->nrange().elements();
```

Note this only works after V3Width.